tests/library/inspector/recorder-api.spec.ts

```diff
@@ -59,7 +59,7 @@ test('should click', async ({ context }) => {
         name: 'click',
         selector: 'internal:role=button[name="Submit"i]',
         ref: 'e2',
-        ariaSnapshot: '- button "Submit" [active] [ref=e2]',
+        ariaSnapshot: `- button "Submit"${browserName === 'webkit' ? '' : ' [active]'} [ref=e2]`,
```

Comment:
nit: would you like to avoid a template literal? I find it harder to read.

```suggestion
        ariaSnapshot: browserName === 'webkit' ? '- button "Submit" [ref=e2]' : '- button "Submit" [active] [ref=e2]',
```